src/Services/Page.vala

```diff
@@ -78,6 +86,23 @@ public class ENotes.PageTable : DatabaseTable {
         }
 
         set_table_name ("Page");
+          
+        // @translator this code summarises a notebook page. Instead of given youtube link this code changes into Youtube Video      
+    
+        // Explaination for link: Regex for [Something](Something). As greedy as editor on markdown
+        var link = new BLMember(/\[[a-zA-Z0-9_\.\?\/:\=\+&\-'"]*\]\([a-zA-Z0-9_\.\?\/:\=\+&\-'"]*\)/, _("Link"));
+       
+        //  \[\^\d+\]:? leads to e.g. [^32], [^68]: 
+        var anchor = new BLMember (/\[\^\d+\]:?/, "");
+    
+        Regex_complex_commands.add (link);
```

Comment:
variable names go `like_this`, without the first capital letter

Reply:
done